packages/blade/src/components/SkipNav/SkipNav.web.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import type { CSSObject } from 'styled-components';
+import styled from 'styled-components';
+import BaseText from '../Typography/BaseText';
+
+export const fallbackId = 'blade-skip-nav';
+// remove once VisuallyHidden is merged
+const screenReaderStyles: CSSObject = {
+  border: 0,
+  clip: 'rect(0 0 0 0)',
+  clipPath: 'inset(50%)',
+  height: '1px',
+  margin: '0 -1px -1px 0',
+  overflow: 'hidden',
+  padding: 0,
+  position: 'absolute',
+  width: '1px',
+  whiteSpace: 'nowrap',
+  wordWrap: 'normal',
+};
+
+type SkipNavLinkProps = {
+  id?: string;
+  children?: string;
+};
+
+const StyledLink = styled.a(({ theme }) => ({
+  ...screenReaderStyles,
+  background: theme.colors.action.background.tertiary.default,
+  top: theme.spacing[6],
+  left: theme.spacing[6],
+  color: theme.colors.action.text.link.default,
+  ':focus': {
+    padding: theme.spacing[2],
+    clip: 'auto',
+    clipPath: 'unset',
+    width: 'auto',
+    height: 'auto',
+  },
+}));
+
+const SkipNavLink = ({
+  id = fallbackId,
+  children = 'Skip to content',
+}: SkipNavLinkProps): JSX.Element => {
+  return (
+    <StyledLink href={`#${id}`}>
+      <BaseText color="action.text.link.default" lineHeight="s">
+        {children}
+      </BaseText>
+    </StyledLink>
+  );
+};
+
+type SkipNavContentProps = {
+  id?: string;
+};
+
+const SkipNavContent = ({ id = fallbackId }: SkipNavContentProps): JSX.Element => {
```

Comment:
this id and link id has to be same right? else this won't work?

Reply:
Yes.

Reply:
since we have default fallback id, imagine I've 2 skipnavcontent and 2 skipnav link how will the navigation work since the id is being set to default fallback

Reply:
That's where you'll add `id` prop in one of them. In most cases there will be one skipnav only. Two skip navs are rare